R/onLoad.R

```diff
@@ -0,0 +1,5 @@
+.onLoad <- function(libname, pkgname) {
+  if (is.null(getOption("rsconnect.max.bundle.size"))) {
+    options(rsconnect.max.bundle.size = 1048576000)
```

Comment:
Also: does that size limit apply to the bundle (compressed) or the app itself (uncompressed)?